server/src/test/java/org/opensearch/cluster/routing/remote/RemoteRoutingTableServiceTests.java

```diff
@@ -105,25 +106,28 @@ public void setup() {
             .build();
         clusterSettings = new ClusterSettings(settings, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS);
         clusterService = mock(ClusterService.class);
+        blobStoreTransferService = mock(BlobStoreTransferService.class);
         when(clusterService.getClusterSettings()).thenReturn(clusterSettings);
         blobStoreRepository = mock(BlobStoreRepository.class);
         when(blobStoreRepository.getCompressor()).thenReturn(new DeflateCompressor());
         blobStore = mock(BlobStore.class);
         blobContainer = mock(BlobContainer.class);
-        when(repositoriesService.repository("routing_repository")).thenReturn(blobStoreRepository);
+        when(repositoriesService.repository(anyString())).thenReturn(blobStoreRepository);
         when(blobStoreRepository.blobStore()).thenReturn(blobStore);
+        when(blobStore.blobContainer(any())).thenReturn(blobContainer);
```

Comment:
Can we add expected blobPath to this instead of `any`

Reply:
This will be used by multiple paths, hence keeping it `any()`

Reply:
Are we verifying the blobPath in individual tests then?

Reply:
Added it in `testGetAsyncIndexRoutingWriteAction`.